modules/op-release-notes-1-7.adoc

```diff
@@ -339,4 +339,24 @@ time="2021-11-04T13:05:26Z" level=error msg="exit status 127"
 // https://github.com/tektoncd/operator/pull/599
 
 
+[id="release-notes-1-7-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.7.1
+
+[id="fixed-issues-1-7-1_{context}"]
+=== Fixed issues
+
+* Before this update, upgrading the {pipelines-title} Operator deleted the data in the database associated with {tekton-hub}, and installed a new database. With this update, an Operator upgrade preserves the data.
```

Comment:
you don't need the comma before `and`:

`...deleted the data in the database associated with {prod} and installed a new database.`